docs/02-get-started/04-trigger-workload-with-event.md

```diff
@@ -167,17 +167,23 @@ The operation was successful if the returned status says `true`.
   Kyma Dashboard
   </summary>
 
-1. Using the left navigation, go back to **Workloads** > **Functions**.
-2. Select your `lastorder` Function and navigate to the **Configuration** tab.
-3. Click on **Add Event Subscription+**.
-4. Provide the following parameters:
-    - **Application name**: `myapp`
+1. Using the left navigation go to **Apllications**.
+2. Click on **Create Application+**.
+3. Provide a name for your application, for example `myapp`.
+4. Go to **Namespaces** and click on `default`.
+5. Using the left navigation, go back to **Workloads** > **Functions**.
+6. Select your `lastorder` Function and navigate to the **Configuration** tab.
+7. Click on **Create Subscription+**.
+8. Provide the following parameters:
+    - **Subscription name**, for example `received-orders`
+    - **Application name**: choose `myapp`
     - **Event name**: `order.received`
     - **Event version**: `v1`
+
     **Event type** is generated automatically; in this example: `myapp.order.received.v1`. 
     
-5. Click **Add**. 
-   The name of the event Subscription is generated automatically and follows the `{FUNCTION_NAME}-{RANDOM_SUFFIX}` pattern.
+9. Click **Create**. 
+   The name of the event Subscription is generated automatically.
```

Comment:
```suggestion
```

It's not any more. You give the name, and it's required. 
![image](https://user-images.githubusercontent.com/53601578/148782772-73d6ac47-999f-40c9-a4d8-e1483e0601d6.png)